webapp/packages/core-authentication/src/UserInfoResource.ts

```diff
@@ -140,6 +140,8 @@ export class UserInfoResource extends CachedDataResource<UserInfo | null, void,
             return authInfo as AuthInfo;
           },
           1000,
+          undefined,
+          300_000, // 5 minutes
```

Comment:
we can use this approach to avoid comments

5 * 60 * 1000

Reply:
fixed